### PR TITLE
Update version and dependencies

### PR DESCRIPTION
**Pull request by buijennifer**
This package is a dependency of `bluesky`.

### Recipe ###
- [x] update version
- [x] update sha
- [x] update dependencies (according to https://github.com/conda-forge/lmfit-feedstock/blob/master/recipe/meta.yaml)
- [x] update recipe maintainers
- [x] sort dependencies

### Package ###
- [x] missing dependencies uploaded to `nsls2forge`
    - [x] `asteval`
    - [x] `uncertainties`

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ package:
 source:
   url: https://github.com/{{ name }}/{{ name }}-py/archive/{{ version }}.tar.gz
   fn: {{ name }}-{{ version }}.tar.gz
-  sha256: cb96919efe57e800cec3187edb88fb8c4aecd0b5687f7fd9d6e4bd10f746880a
+  sha256: b604afca0aaa23b28170065853b59d6e4c585c367e537b769813af313f8ebeed
```
Comment (mrakitin) [inline recipe/meta.yaml]:
```suggestion
  sha256: b604afca0aaa23b28170065853b59d6e4c585c367e537b769813af313f8ebeed
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -18,11 +18,19 @@ build:
 requirements:
   host:
     - python
+    - asteval>=0.9.11
+    - numpy
     - pip
+    - scipy>=0.17
+    - six
+    - uncertainties>=3.0
```
Comment (mrakitin) [inline recipe/meta.yaml]: Will it build without the added dependencies in the `host` section?

Comment (buijennifer) [reply recipe/meta.yaml]: I will test it.

Comment (mrakitin) [reply recipe/meta.yaml]: Thank you.

**Pull request merged**